<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResultsDialog</class>
 <widget class="QDialog" name="ResultsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="labelTitle">
       <property name="text">
        <string>Results: Session XXX</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tabScheduleMain">
          <attribute name="title">
           <string>Main Schedule</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QTableWidget" name="tableMainFreqAll">
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
             <column>
              <property name="text">
               <string>Key</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Count</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Rate</string>
              </property>
             </column>
            </widget>
           </item>
           <item>
            <widget class="QTableWidget" name="tableMainDurAll">
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
             <column>
              <property name="text">
               <string>Key</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Duration</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Percentage</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tabScheduleOne">
          <attribute name="title">
           <string>Schedule 1</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QTableWidget" name="tableMainFreqOne">
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
             <column>
              <property name="text">
               <string>Key</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Count</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Rate</string>
              </property>
             </column>
            </widget>
           </item>
           <item>
            <widget class="QTableWidget" name="tableMainDurOne">
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
             <column>
              <property name="text">
               <string>Key</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Duration</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Percentage</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tabScheduleTwo">
          <attribute name="title">
           <string>Schedule 2</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QTableWidget" name="tableMainFreqTwo">
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
             <column>
              <property name="text">
               <string>Key</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Count</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Rate</string>
              </property>
             </column>
            </widget>
           </item>
           <item>
            <widget class="QTableWidget" name="tableMainDurTwo">
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
             <column>
              <property name="text">
               <string>Key</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Duration</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Percentage</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tabScheduleThree">
          <attribute name="title">
           <string>Schedule 3</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QTableWidget" name="tableMainFreqThree">
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
             <column>
              <property name="text">
               <string>Key</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Count</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Rate</string>
              </property>
             </column>
            </widget>
           </item>
           <item>
            <widget class="QTableWidget" name="tableMainDurThree">
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
             <column>
              <property name="text">
               <string>Key</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Duration</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Percentage</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QTabWidget" name="tabWidget_2">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tableWidgetLog">
          <attribute name="title">
           <string>Session Log</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QListView" name="listView">
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>Cumulative Counts</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item>
            <layout class="QVBoxLayout" name="plotLayout"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>Cumulative Percentage</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_10">
           <item>
            <layout class="QVBoxLayout" name="plotLayout2"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ResultsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ResultsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
